src/content/docs/apm/agents/net-agent/net-agent-api/getbrowsertimingheader-net-agent-api.mdx

```diff
@@ -113,7 +113,39 @@ An HTML string to be embedded in a page header.
   ...
 ```
 
-```razor
+```cshtml
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    @Html.Raw(NewRelic.Api.Agent.NewRelic.GetBrowserTimingHeader("YOUR_NONCE_VALUE"))
+    ...
+  </head>
+  <body>
+  ...
+```
+
+### With Blazor [#blazor]
+
+<Callout variant="important">
+  This API is not supported for Blazor Webassembly, because the agent is unable to instrument Webassembly code.  The following examples are for Blazor Server applications only.  Use the [copy-paste method](https://docs.newrelic.com/docs/browser/browser-monitoring/installation/install-browser-monitoring-agent/#copy-paste-app) of adding the Browser Agent to Blazor Webassembly pages.
+</Callout>
+
+<Callout variant="important">
+  This API can not be placed in a `<HeadContent>` element of a .razor page.  Instead, it should be called from _Layout.cshtml or an equivalent layout file.
```

Comment:
```suggestion
  This API can not be placed in a `<HeadContent>` element of a `.razor` page.  Instead, it should be called from `_Layout.cshtml` or an equivalent layout file.
```

Reply:
@nr-ahemsath Not sure if these should be braced for code block, but added it here

Reply:
That works, thanks for doing that.